RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -290,14 +293,20 @@
 initialStep = ClassifierMerger.clone()
 initialStep.inputClassifiers=['initialStepClassifier1','initialStepClassifier2','initialStepClassifier3']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(initialStep, initialStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorInitialStep_Phase1'),
+     src = 'initialStepTracks',
+     qualityCuts = [-0.95,-0.85,-0.75]
+))
+(trackingPhase1 & fastSim).toModify(initialStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
+
 from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(initialStep, TrackLwtnnClassifier.clone(
-	src = 'initialStepTracks',
-	qualityCuts = [0.0, 0.3, 0.6],
+trackdnn.toReplaceWith(initialStep, TrackLwtnnClassifier.clone(
+        src = 'initialStepTracks',
+        qualityCuts = [0.0, 0.3, 0.6]
 ))
-(trackingPhase1 & fastSim).toModify(initialStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
+(trackdnn & fastSim).toModify(initialStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
 
 pp_on_AA_2018.toReplaceWith(initialStep, initialStepClassifier1.clone( 
      qualityCuts = [-0.9, -0.5, 0.2],
```

Comment:
this was previously simpler, using toModify

Reply:
Changed back to using toModify